dm/dm/proto/dmworker.proto

```diff
@@ -404,3 +406,22 @@ message CheckSubtasksCanUpdateResponse {
   bool success = 1;
   string msg = 2; // message when error occurred
 }
+
+message GetValidationStatusRequest {
+    string taskName = 1;
+    string FilterStatus = 2;
```

Comment:
this field better to use enum

Reply:
This field is already enum in DM (i.e., pb.Stage_Running). Did you mean I should use `Stage FilterStatus = 2;` rather than `string FilterStatus = 2;`?

Reply:
> this field better to use enum

If so, I don't think it works. `FilterStatus` can be empty string which indicates the master should return all the subtasks with either `Running` stage or `Stopped` stage. If using Stage enum here, I can't do this intuitively.

Reply:
if ctl not pass FilterStatua as filter, pb will pass `InvalidStage` as default value, It's a long-standing criticism of pb3

> every enum definition must contain a constant that maps to zero as its first element. 
see [here](https://developers.google.com/protocol-buffers/docs/proto3#enum)

Reply:
btw, why just `FilterStatus`  is Capital letters 🤣

Reply:
Will fix it